reflex/components/radix/themes/layout/stack.py

```diff
@@ -41,12 +41,32 @@ def create(
 class VStack(Stack):
     """A vertical stack component."""
 
-    def _apply_theme(self, theme: Component):
-        self.style.update({"flex_direction": "column"})
+    @classmethod
```

Comment:
Couldn't we just event set it as a prop:

`direction: Var[Literal[...]] = "column"`